index.js

```diff
@@ -19,20 +19,6 @@ class Dropdown extends Component {
     this.fireChangeEvent = this.fireChangeEvent.bind(this)
   }
 
-  componentWillReceiveProps (newProps) {
```

Comment:
I believe this is still needed if the value gets changed outside of the component itself. 

See we have an app like this:

```jsx
<Parent>
  <Child value={value} updateValue={updateValue}/>
  <Dropdown value={value}/>
</Parent>
```

If `<Child>` updates the value, the `Dropdown` will also need to know when the `value` gets updated outside.